### PR TITLE
Fix wrong nesting of _AccessibleButton.scss

### PR DESCRIPTION
**Pull request by luixxiul**
Retry of #8133 

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

@t3chguy the fix should work this time... sorry

Notes: none

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://pr8134--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/elements/_AccessibleButton.scss [hunk 1]
```diff
@@ -16,152 +16,152 @@ limitations under the License.
 
 .mx_AccessibleButton {
     cursor: pointer;
+}
 
-    .mx_AccessibleButton_disabled {
```
Comment (t3chguy) [inline res/css/views/elements/_AccessibleButton.scss:20]: Why not use the `&` selector here?

e.g `&.mx_AccessibleButton_disabled {`

Comment (luixxiul) [reply res/css/views/elements/_AccessibleButton.scss:20]: I thought just reverting to the [latest one](https://github.com/luixxiul/matrix-react-sdk/blob/d5eb4d45dad7b7427607e1dcd10572bfab87603d/res/css/views/elements/_AccessibleButton.scss) before applying SCSS style nesting should work.

> Why not use the `&` selector here?

I think line 21 below should take care of that.

Comment (t3chguy) [reply res/css/views/elements/_AccessibleButton.scss:20]: Yes, just wondering why you're removing the nesting when it can made work

Comment (luixxiul) [reply res/css/views/elements/_AccessibleButton.scss:20]: I guess you're right. I'm pushing another commit.

File: res/css/views/elements/_AccessibleButton.scss [hunk 2]
```diff
@@ -17,151 +17,151 @@ limitations under the License.
...
-        justify-content: center;
-        font-size: $font-14px;
-        border: none; // override default <button /> styles
-
-        .mx_AccessibleButton_kind_primary_sm,
-        .mx_AccessibleButton_kind_danger_sm,
-        .mx_AccessibleButton_kind_link_sm {
-            padding: 5px 12px;
-        }
+.mx_AccessibleButton_hasKind {
```
Comment (t3chguy) [inline res/css/views/elements/_AccessibleButton.scss:34]: The same can be done here

File: res/css/views/elements/_AccessibleButton.scss [hunk 3]
```diff
@@ -17,151 +17,151 @@ limitations under the License.
...
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background-color: #ffffff;
+        mask-repeat: no-repeat;
+        mask-position: center;
+        mask-size: 80%;
```
Comment (t3chguy) [inline res/css/views/elements/_AccessibleButton.scss:165]: And everywhere for that matter, these classes should only be able to be used in conjunction IMO

Comment (luixxiul) [reply res/css/views/elements/_AccessibleButton.scss:165]: Yes, that totally makes sense. Please wait for a minute

**Review by t3chguy — APPROVED** [review]
Great, thanks!!

**Comment by luixxiul** [comment]
@t3chguy thanks for taking your time! I was breaking into a cold sweat pretty much :sweat:

**Pull request merged**